src/test/run-make/tools.mk

```diff
@@ -1,3 +1,6 @@
+# Default to `/tmp` if no other tmp directory is specified.
+TMPDIR ?= "/tmp"
```

Comment:
Hm that's kinda scary, I'd rather leave this out for now as it's the wrong definition on Windows and perhaps investigate later what's going on locally for you?